styles.css

```diff
@@ -268,3 +277,225 @@ textarea {
 }
 
 /* footer end */
+
+/* Desktop Version Start */
+
+@media (min-width: 1440px) {
+  /* desktop Nav */
+  body {
+    background-color: #e5e5e5;
+  }
+
+  .mobile-view {
+    width: auto;
+    background-color: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  nav h2,
+  nav img {
+    display: none;
+  }
+
+  nav {
+    display: block;
+  }
+
+  .destop-menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    list-style: none;
+    padding: 20px 0;
+    margin: 0;
+  }
+
+  .destop-menu-list {
+    margin: 0 30px;
+    font-size: 1.2em;
+  }
+
+  /* Desktop Nav End */
+
+  /* Desktop Header Start */
+  header {
+    background-image: none;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+    gap: 24px;
+    position: absolute;
+    width: 920px;
+    height: 756px;
+    left: 254px;
+    top: 98px;
+  }
+
+  .nav-header-background {
+    background-image: url(images/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    width: 1440px;
+    height: 1250px;
+    top: -398px;
+    left: 27px;
+  }
```

Comment:
Also, kindly adjust the background image so as not to overlap with the social media icons. A closer look at the figma design shows how spaced the images are from the texts.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199489867-02524a8f-3358-4c9a-9b89-9b61077fea8f.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/199489911-d0f63e0b-e643-4da7-afbe-c09e0d47084b.png)

Reply:
Thank you for the review.  Issues fixed.